documentation/user_docs/docs/example_scripts/dnd_vs_sqw.rst

```diff
@@ -0,0 +1,206 @@
+dnd vs sqw: the difference
+##########################
+
+Tutorial to illustrate and explain the different data types in Horace, and why
+they are important.
+
+For more information on the differences between ``dnd`` and ``sqw``, :ref:`see
+here <manual/FAQ:What is the difference between sqw and dnd objects>`
+
+.. code-block:: matlab
```

Comment:
As this is no longer just a script, please add non-script intro to this first section